calicoctl/commands/node/diags.go

```diff
@@ -134,6 +136,9 @@ func runDiags(logDir string) {
 		fmt.Printf("No logs found in %s; skipping log copying", logDir)
 	}
 
+	// Try to copy logs from containers for hosted installs.
+	getDockerLogs(tmpLogDir)
```

Comment:
I think `getHostedLogs` might be more appropriate?  As it is what we are trying to get, WDYT?